frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -6,6 +6,7 @@ import {BoardVersionActionService} from "core-app/modules/boards/board/board-act
 import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/query-updated.service";
 import {BoardAssigneeActionService} from "core-app/modules/boards/board/board-actions/assignee/assignee-action.service";
 import {BoardSubprojectActionService} from "core-app/modules/boards/board/board-actions/subproject/subproject-action.service";
+import {BoardSubtasksActionService} from "core-app/modules/boards/board/board-actions/subtasks/board-subtasks-action.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/8c7032b63674461898491e59e2fb5b6f0ee99baa/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L10"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>